sporkfish/searcher/minimax.py

```diff
@@ -129,7 +129,14 @@ def _aspiration_windows_search(
             )
         return score, move
 
-    def _quiescence(self, board: Board, depth: int, alpha: float, beta: float) -> float:
+    def _quiescence_search(
```

Comment:
Nice rename

[nitpick/oppinion] for me a `search` method would return a `chess.Move`, perhaps `_quiescence_score` would be a better name?

Reply:
Ok, I'll just revert back to the old name later - negamax_score doesn't sound great imo.

Reply:
Reverted